modules/cert-manager-configuring-routes.adoc

```diff
@@ -0,0 +1,150 @@
+  dnsNames:
+    - <hostname> # <3>
+  usages:
+    - server auth
+  issuerRef:
+    kind: Issuer
+    name: letsencrypt-acme
+  secretName: <secret_name> # <4>
+EOF
+----
+<1> Specify the `namespace` where the `Certificate` is located. It should be the same as your route's namespace.
+<2> Specify the certificate's common name using the `hostname` of the route.
+<3> Add the `hostname` of your route to the certificate's DNS names.
+<4> Specify the name of the secret that contains the certificate.
+
+. Create a `Role` to provide the router service account permissions to read the referenced secret using the following command:
++
+[source, terminal]
+----
+$ oc create role secret-reader \
+  --verb=get,list,watch \
+  --resource=secrets \
+  --resource-name=<secret_name> \ # <1>
+  --namespace=<namespace> # <2>
+----
+<1> Specify the name of the secret that you want to grant access to. It should be consistent with your `secretName` specified in the `Certificate`.
+<2> Specify the namespace where both your secret and route reside.
+
+. Create a `RoleBinding` to bind the router service account with the newly created `Role` using the following command:
++
+[source, terminal]
+----
+$ oc create rolebinding secret-reader-binding \
+  --role=secret-reader \
+  --serviceaccount=openshift-ingress:router \
+  --namespace=<namespace> # <1>
+----
+<1> Specify the namespace where both your secret and route reside.
+
+. Update your route's `.spec.tls.externalCertificate` field to reference the previously created secret and use the certificate issued by cert-manager using the following command:
++
+[source, terminal]
+----
+$ oc patch route <route_name> \ # <1>
+  -n <namespace> \ # <2>
+  --type=merge \
+  -p '{"spec":{"tls":{"externalCertificate":{"name":"<secret_name>"}}}}' # <3>
+----
+<1> Specify the route name.
+<2> Specify the namespace where both your secret and route reside.
+<3> Specify the name of the secret that contains the certificate.
+
+.Verification
+
+* Verify that the certificate is created and ready to use by running the following command:
++ 
+[source, terminal]
+----
+$ oc get certificate -n <namespace> <1>
+$ oc get secret -n <namespace> <1>
+----
+<1> Specify the namespace where both your secret and route reside.
+
+* Verify that the router is using the referenced external certificate by running the following command. The command should return with the status code `200 OK`.
++ 
+[source, terminal]
+----
+$ curl -IsS https://<hostname> <1>
+----
+<1> Specify the `hostname` of your route.
+
+* Verify the server certificate's `subject`, `subjectAltName` and `issuer` are all as expected from the curl verbose outputs by running the following command:
++ 
+[source, terminal]
+----
+$ curl -v https://<hostname> <1>
+----
+<1> Specify the `hostname` of your route.
++
+Now the route is successfully secured by the certificate from the referenced secret issued by cert-manager. cert-manager will automatically manage the certificate's lifecycle.
```

Comment:
Should this be attached to the previous (third) bullet? Or should it be decoupled and apply to the overall "Verification" list.

Also suggest moving the "now" in the sentence a little:
```suggestion

The route is now successfully secured by the certificate from the referenced secret issued by cert-manager. cert-manager will automatically manage the certificate's lifecycle.
```